font-inspector.html

```diff
@@ -161,22 +163,32 @@ <h1>Free Software</h1>
 }
 
 function displayNames(names) {
-    var html = '';
-    properties = sortKeys(names);
-    for (var i = 0; i < properties.length; i++) {
-        var property = properties[i];
-        html += '<dt>'+escapeHtml(property)+'</dt><dd>';
-        var translations = names[property];
-        var langs = sortKeys(translations);
-        for (var j = 0; j < langs.length; j++) {
-            var lang = langs[j];
-            var esclang = escapeHtml(lang);
-            html += '<span class="langtag">' + esclang
-                + '</span> <span class="langname" lang=' + esclang + '>'
-                + escapeHtml(translations[lang]) + '</span> ';
+    let html = '';
```

Comment:
Actually, the rest of this is using var which means older browsers (like old iphones etc) will work with it, which do we want to use? We can fix this with a later commit though.

Reply:
@Connum see above.

Reply:
Yeah, you're right, let's not break backwards compatibility where we don't need to.